packages/settingeditor/src/plugineditor.ts

```diff
@@ -205,10 +231,45 @@ namespace PluginEditor {
    */
   export
   interface IOptions {
+    /**
+     * The toolbar commands and registry for the setting editor toolbar.
+     */
+    commands: {
```

Comment:
misaligned and missing new line

Reply:
`commands` is an object that is a child attribute of the `IOptions`